src/Feliz.Plotly/Feliz.Plotly.fsproj

```diff
@@ -5,6 +5,18 @@
     <PackageId>Feliz.Plotly.V2</PackageId>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Version>2.28.0</Version>
+    <PackageReadmeFile>readme.md</PackageReadmeFile>
```

Comment:
You have a readme in the package, but without this NuGet.org won't show it for users.